src/open_inwoner/pdc/tests/test_category.py

```diff
@@ -86,7 +86,7 @@ def test_only_published_subcategories_exist_in_detail_page_when_logged_in(self):
     def test_only_published_categories_exist_in_my_categories_page(self):
         response = self.app.get(reverse("profile:categories"), user=self.user)
         self.assertEqual(
-            list(response.context["categories"]),
+            list(response.context["menu_categories"]),
```

Comment:
This test didn't test what it supposed to test (it is not about the menu but about a form), so this fix is not what we want. I'm rewriting it right now.

Reply:
@Bartvaderkin The bad test is no longer bad - all seems good now! + and @pi-sigma also the "configure Onderwerpen to become invisible for logged out users" still seems to be working this way. May have something to do with the 'context'.